quotefault/__init__.py

```diff
@@ -374,6 +374,15 @@ def hidden():
         metadata=metadata
     )
 
+@app.route('/random', methods=['GET'])
+@auth.oidc_auth
+def random_quote():
+  quote = db.session.execute("SELECT * FROM quotefault.quote WHERE hidden=0 ORDER BY RAND() LIMIT 1;").all()[0]
+  #quote = get_quote_query(speaker = request.args.get('speaker'),
+  #      submitter = request.args.get('submitter')).order_by(func.rand()).limit(1).all()[0][0]

```

Comment:
It might be nice to leave the query args on this route, otherwise please remove the comment.